data/korath/korath ships.txt

```diff
@@ -383,89 +383,845 @@ ship "'olofez"
+ship "Lor'kas Ik 577"
 	sprite "ship/world-ship a"
 	thumbnail "thumbnail/world-ship a"
 	attributes
-		category "Utility"
-		cost 27690000
-		shields 47000
-		hull 34000
-		"required crew" 794
-		"bunks" 1492
+		category "Heavy Freighter"
+		cost 28650000
+		shields 32700
+		hull 29600
+		"required crew" 234
+		"bunks" 453
+		"mass" 1915
+		"drag" 19
+		"heat dissipation" .4
+		"fuel capacity" 1000
+		"cargo space" 1375
+		"outfit space" 883
+		"weapon capacity" 338
+		"engine capacity" 237
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Banisher Grav-Turret" 2
+		"Warder Anti-Missile" 4
+		"Grab-Strike Turret" 2
+		"Expeller Grav-Ray" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Medium)"
+		"Large Heat Shunt" 2
+		"Fuel Processor"
+		"Thermal Repeater Rifle" 237
+		"Microbot Defense Station" 17
+		
+		"Thruster (Stellar Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+	
+	engine -37 333 .8
+	engine 0 333 .8
+	engine 37 333 .8
+	"reverse engine" -19 -303 0.5
+		over
+	"reverse engine" 0 -303 0.5
+		over
+	"reverse engine" 19 -303 0.5
+		over
+	gun -47 -244 "Expeller Grav-Ray"
+	gun 47 -244 "Expeller Grav-Ray"
+	turret -73 -159 "Warder Anti-Missile"
+	turret 73 -159 "Grab-Strike Turret"
+	turret -56 -137 "Warder Anti-Missile"
+	turret 56 -137 "Warder Anti-Missile"
+	turret -58 -6 "Banisher Grav-Turret"
+	turret 58 -6 "Banisher Grav-Turret"
+	turret -58 25 "Grab-Strike Turret"
+	turret 58 25 "Warder Anti-Missile"
+	fighter -72 294 back
+	fighter 0 294 back
+	fighter 72 294 back
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This simple ship has been adapted for hauling materials between refinery and factory stations."
+
+
+ship "Ra'gru Ik 618"
+	sprite "ship/world-ship b"
+	thumbnail "thumbnail/world-ship b"
+	attributes
+		category "Heavy Warship"
+		cost 37640000
+		shields 51000
+		hull 39500
+		"required crew" 619
+		"bunks" 794
 		"mass" 1735
 		"drag" 21
-		"heat dissipation" .4
+		"heat dissipation" .48
 		"fuel capacity" 1000
-		"cargo space" 264
-		"outfit space" 839
-		"weapon capacity" 314
-		"engine capacity" 165
+		"cargo space" 191
+		"outfit space" 1278
+		"weapon capacity" 633
+		"engine capacity" 263
 		weapon
 			"blast radius" 400
 			"shield damage" 8000
 			"hull damage" 4000
 			"hit force" 12000
 	outfits
 		"Grab-Strike Turret" 2
+		"Banisher Grav-Turret" 2
+		"Warder Anti-Missile" 2
+		"Fire-Lance" 3
+		"Korath Inferno" 2
+		"Blaze-Pike Turret" 2
+		"Firelight Missile Bank" 2
+		"Firelight Missile" 36
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Fuel Processor"
+		"Brig"
+		"Microbot Defense Station" 23
+		"Thermal Repeater Rifle" 359
```

Comment:
```suggestion
		"Thermal Repeater Rifle" 794
```

Given that this is the warship variant, and specifically built for combat and potentially raiding, I think it would make sense for it to have an offensive rifle for every required crew, with extras for temporary crew.